cmd/users/main.go

```diff
@@ -313,28 +319,70 @@ func newService(db *sqlx.DB, tracer opentracing.Tracer, auth mainflux.AuthServic
 			Help:      "Total duration of requests in microseconds.",
 		}, []string{"method"}),
 	)
-	if err := createAdmin(svc, userRepo, c); err != nil {
+	if err := createAdmin(svc, userRepo, c, auth); err != nil {
 		logger.Error("failed to create admin user: " + err.Error())
 		os.Exit(1)
 	}
+
+	if c.adminCreatesUser != "" {
```

Comment:
It will be better if `c.adminCreatesUser` is compare with some constant string instead of comparing with `""` empty string
Example :
```suggestion
	if c.adminCreatesUser != "true" {
```

Reply:
So if admin only need to create users , 
then variable `MF_ONLY_ADMIN_CREATES_USER` should set to `true`
Or Can be make its as `Boolean`.?

Reply:
The user configures the environment variables on the `.env` file and can write any arbitrary string as a value for a particular environment variable. I think, forcing a user to write a fixed value for the environment variable is not convenient. Of course, both ways will work in the end.

What I've done here is that if the user writes any value for the MF_ONLY_ADMIN_CREATES_USER (which eventually makes `c.adminCreatesUser != ""` true), it means that the only admin is capable to create new users.